image.yaml

```diff
@@ -1,9 +1,9 @@
 schema_version: 1
 
-name: "jboss-eap-7/eap72-openshift"
+name: "jboss-eap-7-beta/eap72-openshift"
 description: "Red Hat JBoss Enterprise Application Platform 7.2 OpenShift container image"
 version: "1.0"
-from: "jboss-eap-7/eap72:latest"
+from: "jboss-eap-7-beta/eap72:latest"
```

Comment:
Something to consider here. For the non CD product, the image will always track the underlying product version, so you can say:

from: "jboss-eap-7-beta/eap72:7.2.0" 

for example. The the latest build of the standalone with that version will always be used.

:latest on the other hand, could be floating, and someone has build a -cp build and overwritten it.